scss/_patterns_contextual-menu.scss

```diff
@@ -0,0 +1,117 @@
+@import 'settings';
+
+@mixin vf-p-contextual-menu {
+  .p-contextual-menu {
+    display: inline-block;
+    margin: 0;
+    position: relative;
+
+    // Dropdown element for contextual menu
+    &__dropdown {
+      @extend .p-card--highlighted; // sass-lint:disable-line placeholder-in-extend
+      display: none;
+      margin: 0;
+      max-width: 330px;
+      min-width: 155px;
+      padding: 0;
+      position: absolute;
+      right: 0;
+      top: calc(100% + #{$sp-x-small});
+      z-index: 1;
+
+      &::before,
+      &::after {
+        border: {
+          bottom: 8px solid transparentize($color-dark, .95);
+          left: 8px solid transparent;
+          right: 8px solid transparent;
+        }
+        bottom: 100%;
+        content: '';
+        height: 0;
+        pointer-events: none;
+        position: absolute;
+        right: $sp-small;
+        width: 0;
+      }
+
+      &::after {
+        border: {
+          bottom: 6px solid $color-x-light;
+          left: 6px solid transparent;
+          right: 6px solid transparent;
+        }
+        right: $sp-small + .1rem;
+      }
+
+      // When set to false will show the contextual menu
+      &[aria-hidden="false"] {
+        display: block;
+      }
+    }
+
+    &__group {
+      display: block;
+      padding: $sp-xx-small 0;
+
+      + .p-contextual-menu__group {
+        border-top: 1px solid $color-mid-light;
+        margin: 0;
+      }
+    }
+
+    &__link {
+      border: 0;
+      clear: both;
+      color: $color-dark;
+      display: block;
+      line-height: 1.5rem;
+      margin: 0;
+      overflow: hidden;
+      padding: $sp-xx-small $sp-small;
+      text-align: left;
+      text-overflow: ellipsis;
+      white-space: nowrap;
+      width: 100%;
+
+      &:hover {
+        background-color: $color-light;
+        text-decoration: none;
+      }
+    }
+
+    // Alignment extension to align the menu to the left
+    &--left {
+
+      .p-contextual-menu__dropdown {
+        left: 0;
+
+        &::before,
+        &::after {
+          left: $sp-small;
+          right: initial;
+        }
+
+        &::after {
+          left: $sp-small + .1rem;
```

Comment:
I don't think there is a size that matches what design wanted hence why I did the said value.